students/g_rama/lesson07/html_render.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+class Element:
+    tag = ""
+    indent = " "
+
+    def __init__(self, content=None, **kwargs):
+        self.indent = 4
+        if content:
+            self.content = [content]
+        else:
+            self.content = []
+        self.html_options = kwargs
+
+    def append(self, p_content):
+        return self.content.append(p_content)
+
+    def render(self, file_out, cur_ind=" "):
+        file_out.write(f"<{self.tag}")
+        for k in self.html_options:
+            file_out.write(f" {k}={self.html_options[k]}")
+        file_out.write(f"> ")
+        file_out.write("\n")
+
+        for _ in range(len(self.content)):
```

Comment:
have you read my previous feedback? `_` is not something you should use for *variable* names.

your loop is also incorrect here, instead should be:
```
for item in self.content:
    item.render(file_out)
```